src/modules/core/components/FileUpload/__tests__/__snapshots__/FileUpload.test.tsx.snap

```diff
@@ -155,7 +155,6 @@ exports[`FileUpload component Renders initial component 1`] = `
               </InputLabel>
               <div
                 className=""
-                data-test="fileUpload"
```

Comment:
Why are you removing this data-test? I am surprised that you can push with it. I had to specifically add  it after adding `data-test` prop to the FileUpload, otherwise it wouldn't pass the push tests.

Reply:
In my PR I set dataTest to a dynamic value `data-test={dataTest}`. Then when regenerating the snapshot it removed it for this reason. I did a search in the `cypress` directory to see if `fileUpload` was being used but could not find it used anywhere.

Reply:
@chinins I'm happy to change it to a static data-test attribute value in FuleUpload. What are your thoughts?

Reply:
you are right, I actually removed it as I changed to using a different selector. Keep it like this - it's all good.